src/main/java/appeng/tile/storage/TileChest.java

```diff
@@ -89,6 +87,8 @@
 import appeng.capabilities.Capabilities;
 import appeng.helpers.IPriorityHost;
 import appeng.me.GridAccessException;
+import appeng.me.helpers.MachineSource;
+import appeng.me.helpers.PlayerSource;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'appeng.me.helpers.PlayerSource'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AUselessImportCheck)